pkg/cli/credential/azure_credential_management.go

```diff
@@ -45,39 +45,36 @@ type AzureCredentialManagementClientInterface interface {
 	Delete(ctx context.Context, name string) (bool, error)
 }
 
-const (
-	AzureCredential     = "azure"
-	AzurePlaneName      = "azurecloud"
-	azureCredentialKind = "ServicePrincipal"
-)
+type AzureCredentialProperties struct {
```

Comment:
Add docs if these are used outside the package. Otherwise, make them non-exported